recovery/translation_de.ts

```diff
@@ -11,42 +11,42 @@
     <message>
         <location filename="backupthread.cpp" line="106"/>
         <source>Cannot backup </source>
-        <translation type="unfinished"></translation>
+        <translation>Backup kann nicht erstellt werden</translation>
     </message>
     <message>
         <location filename="backupthread.cpp" line="106"/>
         <source> :BTRFS file format</source>
-        <translation type="unfinished"></translation>
+        <translation> :BTRFS Dateiformat</translation>
     </message>
     <message>
         <location filename="backupthread.cpp" line="137"/>
         <source>%1: Updating partitions.json</source>
-        <translation type="unfinished"></translation>
+        <translation>partitions.json wird aktualisiert</translation>
```

Comment:
Also missing `%1` ?

Reply:
@procount Although I guess if you made it `%1: Updating %2` then you'd only need a single string-translation for both the `partitions.json` and `os.json` lines :slightly_smiling_face: (and the filenames themselves don't need translating anyway)

Reply:
That is very true. I'll try and remember to pick that up as I go through all the translations according to your previous comments on the use of .arg. :wink: